### PR TITLE
helm chart - Allow cloud config path for AWS provider

### PR DESCRIPTION
**Pull request by khizunov**
#### Which component this PR applies to?
helm charts

#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:

Allow AWS cloud provider to be configured with configuration file e.g endpoints

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by gjtempleton** [comment]
Hey,  thanks for the PR, a couple of things to address though:

1. Can you please bump the version by a minor as we're adding new functionality?
1. Can we add some docs as to what the example being added shows/how it would be used and split it out into separate objects, as it wasn't clear on an initial read that you were providing both example values for the helm chart and the example config map for the cloud-config.

**Comment by khizunov** [comment]
@gjtempleton done 👍

File: cluster-autoscaler/cloudprovider/aws/README.md [hunk 1]
```diff
@@ -421,6 +421,42 @@ If you want to update the vendored AWS SDK to a newer version, please make sure
...
+    extraVolumes:
+      - name: cloud-config
+        configMap:
+          name: cloud-config
+    
+    extraVolumeMounts:
+      - name: cloud-config
+        mountPath: config
+    ```
+3. Install (or upgrade) helm chart with updated values
```
Comment (gjtempleton) [inline cluster-autoscaler/cloudprovider/aws/README.md:445]: Potentially:
```suggestion
3. Install (or upgrade) helm chart with updated values, see [an example here.](examples/values-configmap-example.yaml)
```

Comment (khizunov) [reply cluster-autoscaler/cloudprovider/aws/README.md]: @gjtempleton thanks for suggestion, done

File: charts/cluster-autoscaler/templates/deployment.yaml [hunk 2]
```diff
@@ -63,6 +63,9 @@ spec:
             {{- if .Values.autoDiscovery.clusterName }}
             - --node-group-auto-discovery=asg:tag={{ tpl (join "," .Values.autoDiscovery.tags) . }}
             {{- end }}
+            {{- if .Values.cloudConfigPath }}
```
Comment (SergiySavarin) [inline charts/cluster-autoscaler/templates/deployment.yaml:66]: @khizunov could you consider to change `cloudConfigPath` default value here https://github.com/kubernetes/autoscaler/blob/master/charts/cluster-autoscaler/values.yaml#L128.
Currently, it affects autoscaler start up if file not present:
```
$ kubectl logs cluster-autoscaler-aws-cluster-autoscaler-7cb9d6f588-9ppbk -n dr-jobs
I0829 12:07:52.318157       1 flags.go:57] FLAG: --add-dir-header="false"
I0829 12:07:52.318217       1 flags.go:57] FLAG: --address=":8085"
I0829 12:07:52.318222       1 flags.go:57] FLAG: --alsologtostderr="false"
I0829 12:07:52.318226       1 flags.go:57] FLAG: --aws-use-static-instance-list="false"
I0829 12:07:52.318229       1 flags.go:57] FLAG: --balance-similar-node-groups="false"
I0829 12:07:52.318232       1 flags.go:57] FLAG: --balancing-ignore-label="[]"
I0829 12:07:52.318235       1 flags.go:57] FLAG: --cloud-config="/etc/gce.conf"
I0829 12:07:52.318238       1 flags.go:57] FLAG: --cloud-provider="aws"
I0829 12:07:52.318241       1 flags.go:57] FLAG: --cloud-provider-gce-l7lb-src-cidrs="130.211.0.0/22,35.191.0.0/16"
I0829 12:07:52.318246       1 flags.go:57] FLAG: --cloud-provider-gce-lb-src-cidrs="130.211.0.0/22,209.85.152.0/22,209.85.204.0/22,35.191.0.0/16"
I0829 12:07:52.318251       1 flags.go:57] FLAG: --cluster-name=""
I0829 12:07:52.318254       1 flags.go:57] FLAG: --clusterapi-cloud-config-authoritative="false"
I0829 12:07:52.318257       1 flags.go:57] FLAG: --cordon-node-before-terminating="false"
I0829 12:07:52.318260       1 flags.go:57] FLAG: --cores-total="0:320000"
I0829 12:07:52.318263       1 flags.go:57] FLAG: --daemonset-eviction-for-empty-nodes="false"
I0829 12:07:52.318265       1 flags.go:57] FLAG: --daemonset-eviction-for-occupied-nodes="true"
I0829 12:07:52.318268       1 flags.go:57] FLAG: --emit-per-nodegroup-metrics="false"
I0829 12:07:52.318272       1 flags.go:57] FLAG: --estimator="binpacking"
I0829 12:07:52.318276       1 flags.go:57] FLAG: --expander="random"
I0829 12:07:52.318280       1 flags.go:57] FLAG: --expendable-pods-priority-cutoff="-10"
I0829 12:07:52.318285       1 flags.go:57] FLAG: --feature-gates=""
I0829 12:07:52.318292       1 flags.go:57] FLAG: --gce-concurrent-refreshes="1"
I0829 12:07:52.318297       1 flags.go:57] FLAG: --gpu-total="[]"
I0829 12:07:52.318301       1 flags.go:57] FLAG: --ignore-daemonsets-utilization="false"
I0829 12:07:52.318306       1 flags.go:57] FLAG: --ignore-mirror-pods-utilization="false"
I0829 12:07:52.318310       1 flags.go:57] FLAG: --ignore-taint="[]"
I0829 12:07:52.318313       1 flags.go:57] FLAG: --kubeconfig=""
I0829 12:07:52.318316       1 flags.go:57] FLAG: --kubernetes=""
I0829 12:07:52.318318       1 flags.go:57] FLAG: --leader-elect="true"
I0829 12:07:52.318323       1 flags.go:57] FLAG: --leader-elect-lease-duration="15s"
I0829 12:07:52.318327       1 flags.go:57] FLAG: --leader-elect-renew-deadline="10s"
I0829 12:07:52.318331       1 flags.go:57] FLAG: --leader-elect-resource-lock="leases"
I0829 12:07:52.318335       1 flags.go:57] FLAG: --leader-elect-resource-name="cluster-autoscaler"
I0829 12:07:52.318339       1 flags.go:57] FLAG: --leader-elect-resource-namespace=""
I0829 12:07:52.318343       1 flags.go:57] FLAG: --leader-elect-retry-period="2s"
I0829 12:07:52.318348       1 flags.go:57] FLAG: --log-backtrace-at=":0"
I0829 12:07:52.318356       1 flags.go:57] FLAG: --log-dir=""
I0829 12:07:52.318360       1 flags.go:57] FLAG: --log-file=""
I0829 12:07:52.318365       1 flags.go:57] FLAG: --log-file-max-size="1800"
I0829 12:07:52.318370       1 flags.go:57] FLAG: --logtostderr="true"
I0829 12:07:52.318374       1 flags.go:57] FLAG: --max-autoprovisioned-node-group-count="15"
I0829 12:07:52.318379       1 flags.go:57] FLAG: --max-bulk-soft-taint-count="10"
I0829 12:07:52.318383       1 flags.go:57] FLAG: --max-bulk-soft-taint-time="3s"
I0829 12:07:52.318388       1 flags.go:57] FLAG: --max-empty-bulk-delete="10"
I0829 12:07:52.318392       1 flags.go:57] FLAG: --max-failing-time="15m0s"
I0829 12:07:52.318397       1 flags.go:57] FLAG: --max-graceful-termination-sec="600"
I0829 12:07:52.318401       1 flags.go:57] FLAG: --max-inactivity="10m0s"
I0829 12:07:52.318406       1 flags.go:57] FLAG: --max-node-provision-time="15m0s"
I0829 12:07:52.318411       1 flags.go:57] FLAG: --max-nodes-total="0"
I0829 12:07:52.318414       1 flags.go:57] FLAG: --max-total-unready-percentage="45"
I0829 12:07:52.318418       1 flags.go:57] FLAG: --memory-total="0:6400000"
I0829 12:07:52.318423       1 flags.go:57] FLAG: --min-replica-count="0"
I0829 12:07:52.318425       1 flags.go:57] FLAG: --namespace="dr-jobs"
I0829 12:07:52.318429       1 flags.go:57] FLAG: --new-pod-scale-up-delay="0s"
I0829 12:07:52.318432       1 flags.go:57] FLAG: --node-autoprovisioning-enabled="false"
I0829 12:07:52.318435       1 flags.go:57] FLAG: --node-deletion-delay-timeout="2m0s"
I0829 12:07:52.318438       1 flags.go:57] FLAG: --node-group-auto-discovery="[asg:tag=k8s.io/cluster-autoscaler/enabled,k8s.io/cluster-autoscaler/stackbot-shrink-qu86wrctuxqrkj-EKSCluster]"
I0829 12:07:52.318456       1 flags.go:57] FLAG: --nodes="[]"
I0829 12:07:52.318460       1 flags.go:57] FLAG: --ok-total-unready-count="3"
I0829 12:07:52.318463       1 flags.go:57] FLAG: --one-output="false"
I0829 12:07:52.318466       1 flags.go:57] FLAG: --profiling="false"
I0829 12:07:52.318469       1 flags.go:57] FLAG: --regional="false"
I0829 12:07:52.318473       1 flags.go:57] FLAG: --scale-down-candidates-pool-min-count="50"
I0829 12:07:52.318477       1 flags.go:57] FLAG: --scale-down-candidates-pool-ratio="0.1"
I0829 12:07:52.318484       1 flags.go:57] FLAG: --scale-down-delay-after-add="10m0s"
I0829 12:07:52.318488       1 flags.go:57] FLAG: --scale-down-delay-after-delete="0s"
I0829 12:07:52.318493       1 flags.go:57] FLAG: --scale-down-delay-after-failure="3m0s"
I0829 12:07:52.318499       1 flags.go:57] FLAG: --scale-down-enabled="true"
I0829 12:07:52.318503       1 flags.go:57] FLAG: --scale-down-gpu-utilization-threshold="0.5"
I0829 12:07:52.318508       1 flags.go:57] FLAG: --scale-down-non-empty-candidates-count="30"
I0829 12:07:52.318512       1 flags.go:57] FLAG: --scale-down-unneeded-time="10m0s"
I0829 12:07:52.318516       1 flags.go:57] FLAG: --scale-down-unready-time="20m0s"
I0829 12:07:52.318519       1 flags.go:57] FLAG: --scale-down-utilization-threshold="0.5"
I0829 12:07:52.318522       1 flags.go:57] FLAG: --scale-up-from-zero="true"
I0829 12:07:52.318526       1 flags.go:57] FLAG: --scan-interval="10s"
I0829 12:07:52.318529       1 flags.go:57] FLAG: --skip-headers="false"
I0829 12:07:52.318532       1 flags.go:57] FLAG: --skip-log-headers="false"
I0829 12:07:52.318535       1 flags.go:57] FLAG: --skip-nodes-with-local-storage="true"
I0829 12:07:52.318538       1 flags.go:57] FLAG: --skip-nodes-with-system-pods="true"
I0829 12:07:52.318541       1 flags.go:57] FLAG: --status-config-map-name="cluster-autoscaler-status"
I0829 12:07:52.318545       1 flags.go:57] FLAG: --stderrthreshold="0"
I0829 12:07:52.318548       1 flags.go:57] FLAG: --unremovable-node-recheck-timeout="5m0s"
I0829 12:07:52.318551       1 flags.go:57] FLAG: --user-agent="cluster-autoscaler"
I0829 12:07:52.318554       1 flags.go:57] FLAG: --v="4"
I0829 12:07:52.318557       1 flags.go:57] FLAG: --vmodule=""
I0829 12:07:52.318560       1 flags.go:57] FLAG: --write-status-configmap="true"
I0829 12:07:52.318565       1 main.go:401] Cluster Autoscaler 1.23.0
I0829 12:07:52.332410       1 leaderelection.go:248] attempting to acquire leader lease dr-jobs/cluster-autoscaler...
I0829 12:07:52.347246       1 leaderelection.go:258] successfully acquired lease dr-jobs/cluster-autoscaler
I0829 12:07:52.347387       1 event_sink_logging_wrapper.go:48] Event(v1.ObjectReference{Kind:"Lease", Namespace:"dr-jobs", Name:"cluster-autoscaler", UID:"3c9601df-a127-455c-97a2-419f52d0c1bc", APIVersion:"coordination.k8s.io/v1", ResourceVersion:"377704", FieldPath:""}): type: 'Normal' reason: 'LeaderElection' cluster-autoscaler-aws-cluster-autoscaler-7cb9d6f588-9ppbk became leader
I0829 12:07:52.348571       1 reflector.go:219] Starting reflector *v1.Pod (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:188
I0829 12:07:52.348593       1 reflector.go:255] Listing and watching *v1.Pod from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:188
I0829 12:07:52.348604       1 reflector.go:219] Starting reflector *v1.Node (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:246
I0829 12:07:52.348615       1 reflector.go:255] Listing and watching *v1.Node from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:246
I0829 12:07:52.348639       1 reflector.go:219] Starting reflector *v1.Pod (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:212
I0829 12:07:52.348649       1 reflector.go:255] Listing and watching *v1.Pod from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:212
I0829 12:07:52.348767       1 reflector.go:219] Starting reflector *v1.Node (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:246
I0829 12:07:52.348775       1 reflector.go:219] Starting reflector *v1.ReplicationController (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:329
I0829 12:07:52.348777       1 reflector.go:255] Listing and watching *v1.Node from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:246
I0829 12:07:52.348784       1 reflector.go:255] Listing and watching *v1.ReplicationController from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:329
I0829 12:07:52.348797       1 reflector.go:219] Starting reflector *v1beta1.PodDisruptionBudget (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:309
I0829 12:07:52.348803       1 reflector.go:255] Listing and watching *v1beta1.PodDisruptionBudget from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:309
I0829 12:07:52.348886       1 reflector.go:219] Starting reflector *v1.DaemonSet (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:320
I0829 12:07:52.348894       1 reflector.go:255] Listing and watching *v1.DaemonSet from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:320
I0829 12:07:52.348900       1 reflector.go:219] Starting reflector *v1.ReplicaSet (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:347
I0829 12:07:52.348909       1 reflector.go:255] Listing and watching *v1.ReplicaSet from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:347
I0829 12:07:52.348965       1 reflector.go:219] Starting reflector *v1.Job (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:338
I0829 12:07:52.348973       1 reflector.go:255] Listing and watching *v1.Job from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:338
I0829 12:07:52.348994       1 reflector.go:219] Starting reflector *v1.StatefulSet (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:356
I0829 12:07:52.349001       1 reflector.go:255] Listing and watching *v1.StatefulSet from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:356
W0829 12:07:52.364627       1 warnings.go:70] policy/v1beta1 PodDisruptionBudget is deprecated in v1.21+, unavailable in v1.25+; use policy/v1 PodDisruptionBudget
I0829 12:07:52.364817       1 cloud_provider_builder.go:29] Building aws cloud provider.
F0829 12:07:52.364869       1 aws_cloud_provider.go:352] Couldn't open cloud provider configuration /etc/gce.conf: &fs.PathError{Op:"open", Path:"/etc/gce.conf", Err:0x2}
goroutine 30 [running]:
k8s.io/klog/v2.stacks(0x1)
    /gopath/src/k8s.io/autoscaler/cluster-autoscaler/vendor/k8s.io/klog/v2/klog.go:1038 +0x8a
k8s.io/klog/v2.(*loggingT).output(0x611e4e0, 0x3, 0x0, 0xc000565730, 0x0, {0x4d2e584, 0x2}, 0xc0009ea840, 0x0)
    /gopath/src/k8s.io/autoscaler/cluster-autoscaler/vendor/k8s.io/klog/v2/klog.go:987 +0x5fd
k8s.io/klog/v2.(*loggingT).printf(0...
$ kubectl logs cluster-autoscaler-aws-cluster-autoscaler-7cb9d6f588-9ppbk -n dr-jobs
I0829 12:07:52.318157       1 flags.go:57] FLAG: --add-dir-header="false"
I0829 12:07:52.318217       1 flags.go:57] FLAG: --address=":8085"
I0829 12:07:52.318222       1 flags.go:57] FLAG: --alsologtostderr="false"
I0829 12:07:52.318226       1 flags.go:57] FLAG: --aws-use-static-instance-list="false"
I0829 12:07:52.318229       1 flags.go:57] FLAG: --balance-similar-node-groups="false"
I0829 12:07:52.318232       1 flags.go:57] FLAG: --balancing-ignore-label="[]"
I0829 12:07:52.318235       1 flags.go:57] FLAG: --cloud-config="/etc/gce.conf"
I0829 12:07:52.318238       1 flags.go:57] FLAG: --cloud-provider="aws"
I0829 12:07:52.318241       1 flags.go:57] FLAG: --cloud-provider-gce-l7lb-src-cidrs="130.211.0.0/22,35.191.0.0/16"
I0829 12:07:52.318246       1 flags.go:57] FLAG: --cloud-provider-gce-lb-src-cidrs="130.211.0.0/22,209.85.152.0/22,209.85.204.0/22,35.191.0.0/16"
I0829 12:07:52.318251       1 flags.go:57] FLAG: --cluster-name=""
I0829 12:07:52.318254       1 flags.go:57] FLAG: --clusterapi-cloud-config-authoritative="false"
I0829 12:07:52.318257       1 flags.go:57] FLAG: --cordon-node-before-terminating="false"
I0829 12:07:52.318260       1 flags.go:57] FLAG: --cores-total="0:320000"
I0829 12:07:52.318263       1 flags.go:57] FLAG: --daemonset-eviction-for-empty-nodes="false"
I0829 12:07:52.318265       1 flags.go:57] FLAG: --daemonset-eviction-for-occupied-nodes="true"
I0829 12:07:52.318268       1 flags.go:57] FLAG: --emit-per-nodegroup-metrics="false"
I0829 12:07:52.318272       1 flags.go:57] FLAG: --estimator="binpacking"
I0829 12:07:52.318276       1 flags.go:57] FLAG: --expander="random"
I0829 12:07:52.318280       1 flags.go:57] FLAG: --expendable-pods-priority-cutoff="-10"
I0829 12:07:52.318285       1 flags.go:57] FLAG: --feature-gates=""
I0829 12:07:52.318292       1 flags.go:57] FLAG: --gce-concurrent-refreshes="1"
I0829 12:07:52.318297       1 flags.go:57] FLAG: --gpu-total="[]"
I0829 12:07:52.318301       1 flags.go:57] FLAG: --ignore-daemonsets-utilization="false"
I0829 12:07:52.318306       1 flags.go:57] FLAG: --ignore-mirror-pods-utilization="false"
I0829 12:07:52.318310       1 flags.go:57] FLAG: --ignore-taint="[]"
I0829 12:07:52.318313       1 flags.go:57] FLAG: --kubeconfig=""
I0829 12:07:52.318316       1 flags.go:57] FLAG: --kubernetes=""
I0829 12:07:52.318318       1 flags.go:57] FLAG: --leader-elect="true"
I0829 12:07:52.318323       1 flags.go:57] FLAG: --leader-elect-lease-duration="15s"
I0829 12:07:52.318327       1 flags.go:57] FLAG: --leader-elect-renew-deadline="10s"
I0829 12:07:52.318331       1 flags.go:57] FLAG: --leader-elect-resource-lock="leases"
I0829 12:07:52.318335       1 flags.go:57] FLAG: --leader-elect-resource-name="cluster-autoscaler"
I0829 12:07:52.318339       1 flags.go:57] FLAG: --leader-elect-resource-namespace=""
I0829 12:07:52.318343       1 flags.go:57] FLAG: --leader-elect-retry-period="2s"
I0829 12:07:52.318348       1 flags.go:57] FLAG: --log-backtrace-at=":0"
I0829 12:07:52.318356       1 flags.go:57] FLAG: --log-dir=""
I0829 12:07:52.318360       1 flags.go:57] FLAG: --log-file=""
I0829 12:07:52.318365       1 flags.go:57] FLAG: --log-file-max-size="1800"
I0829 12:07:52.318370       1 flags.go:57] FLAG: --logtostderr="true"
I0829 12:07:52.318374       1 flags.go:57] FLAG: --max-autoprovisioned-node-group-count="15"
I0829 12:07:52.318379       1 flags.go:57] FLAG: --max-bulk-soft-taint-count="10"
I0829 12:07:52.318383       1 flags.go:57] FLAG: --max-bulk-soft-taint-time="3s"
I0829 12:07:52.318388       1 flags.go:57] FLAG: --max-empty-bulk-delete="10"
I0829 12:07:52.318392       1 flags.go:57] FLAG: --max-failing-time="15m0s"
I0829 12:07:52.318397       1 flags.go:57] FLAG: --max-graceful-termination-sec="600"
I0829 12:07:52.318401       1 flags.go:57] FLAG: --max-inactivity="10m0s"
I0829 12:07:52.318406       1 flags.go:57] FLAG: --max-node-provision-time="15m0s"
I0829 12:07:52.318411       1 flags.go:57] FLAG: --max-nodes-total="0"
I0829 12:07:52.318414       1 flags.go:57] FLAG: --max-total-unready-percentage="45"
I0829 12:07:52.318418       1 flags.go:57] FLAG: --memory-total="0:6400000"
I0829 12:07:52.318423       1 flags.go:57] FLAG: --min-replica-count="0"
I0829 12:07:52.318425       1 flags.go:57] FLAG: --namespace="dr-jobs"
I0829 12:07:52.318429       1 flags.go:57] FLAG: --new-pod-scale-up-delay="0s"
I0829 12:07:52.318432       1 flags.go:57] FLAG: --node-autoprovisioning-enabled="false"
I0829 12:07:52.318435       1 flags.go:57] FLAG: --node-deletion-delay-timeout="2m0s"
I0829 12:07:52.318438       1 flags.go:57] FLAG: --node-group-auto-discovery="[asg:tag=k8s.io/cluster-autoscaler/enabled,k8s.io/cluster-autoscaler/stackbot-shrink-qu86wrctuxqrkj-EKSCluster]"
I0829 12:07:52.318456       1 flags.go:57] FLAG: --nodes="[]"
I0829 12:07:52.318460       1 flags.go:57] FLAG: --ok-total-unready-count="3"
I0829 12:07:52.318463       1 flags.go:57] FLAG: --one-output="false"
I0829 12:07:52.318466       1 flags.go:57] FLAG: --profiling="false"
I0829 12:07:52.318469       1 flags.go:57] FLAG: --regional="false"
I0829 12:07:52.318473       1 flags.go:57] FLAG: --scale-down-candidates-pool-min-count="50"
I0829 12:07:52.318477       1 flags.go:57] FLAG: --scale-down-candidates-pool-ratio="0.1"
I0829 12:07:52.318484       1 flags.go:57] FLAG: --scale-down-delay-after-add="10m0s"
I0829 12:07:52.318488       1 flags.go:57] FLAG: --scale-down-delay-after-delete="0s"
I0829 12:07:52.318493       1 flags.go:57] FLAG: --scale-down-delay-after-failure="3m0s"
I0829 12:07:52.318499       1 flags.go:57] FLAG: --scale-down-enabled="true"
I0829 12:07:52.318503       1 flags.go:57] FLAG: --scale-down-gpu-utilization-threshold="0.5"
I0829 12:07:52.318508       1 flags.go:57] FLAG: --scale-down-non-empty-candidates-count="30"
I0829 12:07:52.318512       1 flags.go:57] FLAG: --scale-down-unneeded-time="10m0s"
I0829 12:07:52.318516       1 flags.go:57] FLAG: --scale-down-unready-time="20m0s"
I0829 12:07:52.318519       1 flags.go:57] FLAG: --scale-down-utilization-threshold="0.5"
I0829 12:07:52.318522       1 flags.go:57] FLAG: --scale-up-from-zero="true"
I0829 12:07:52.318526       1 flags.go:57] FLAG: --scan-interval="10s"
I0829 12:07:52.318529       1 flags.go:57] FLAG: --skip-headers="false"
I0829 12:07:52.318532       1 flags.go:57] FLAG: --skip-log-headers="false"
I0829 12:07:52.318535       1 flags.go:57] FLAG: --skip-nodes-with-local-storage="true"
I0829 12:07:52.318538       1 flags.go:57] FLAG: --skip-nodes-with-system-pods="true"
I0829 12:07:52.318541       1 flags.go:57] FLAG: --status-config-map-name="cluster-autoscaler-status"
I0829 12:07:52.318545       1 flags.go:57] FLAG: --stderrthreshold="0"
I0829 12:07:52.318548       1 flags.go:57] FLAG: --unremovable-node-recheck-timeout="5m0s"
I0829 12:07:52.318551       1 flags.go:57] FLAG: --user-agent="cluster-autoscaler"
I0829 12:07:52.318554       1 flags.go:57] FLAG: --v="4"
I0829 12:07:52.318557       1 flags.go:57] FLAG: --vmodule=""
I0829 12:07:52.318560       1 flags.go:57] FLAG: --write-status-configmap="true"
I0829 12:07:52.318565       1 main.go:401] Cluster Autoscaler 1.23.0
I0829 12:07:52.332410       1 leaderelection.go:248] attempting to acquire leader lease dr-jobs/cluster-autoscaler...
I0829 12:07:52.347246       1 leaderelection.go:258] successfully acquired lease dr-jobs/cluster-autoscaler
I0829 12:07:52.347387       1 event_sink_logging_wrapper.go:48] Event(v1.ObjectReference{Kind:"Lease", Namespace:"dr-jobs", Name:"cluster-autoscaler", UID:"3c9601df-a127-455c-97a2-419f52d0c1bc", APIVersion:"coordination.k8s.io/v1", ResourceVersion:"377704", FieldPath:""}): type: 'Normal' reason: 'LeaderElection' cluster-autoscaler-aws-cluster-autoscaler-7cb9d6f588-9ppbk became leader
I0829 12:07:52.348571       1 reflector.go:219] Starting reflector *v1.Pod (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:188
I0829 12:07:52.348593       1 reflector.go:255] Listing and watching *v1.Pod from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:188
I0829 12:07:52.348604       1 reflector.go:219] Starting reflector *v1.Node (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:246
I0829 12:07:52.348615       1 reflector.go:255] Listing and watching *v1.Node from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:246
I0829 12:07:52.348639       1 reflector.go:219] Starting reflector *v1.Pod (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:212
I0829 12:07:52.348649       1 reflector.go:255] Listing and watching *v1.Pod from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:212
I0829 12:07:52.348767       1 reflector.go:219] Starting reflector *v1.Node (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:246
I0829 12:07:52.348775       1 reflector.go:219] Starting reflector *v1.ReplicationController (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:329
I0829 12:07:52.348777       1 reflector.go:255] Listing and watching *v1.Node from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:246
I0829 12:07:52.348784       1 reflector.go:255] Listing and watching *v1.ReplicationController from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:329
I0829 12:07:52.348797       1 reflector.go:219] Starting reflector *v1beta1.PodDisruptionBudget (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:309
I0829 12:07:52.348803       1 reflector.go:255] Listing and watching *v1beta1.PodDisruptionBudget from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:309
I0829 12:07:52.348886       1 reflector.go:219] Starting reflector *v1.DaemonSet (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:320
I0829 12:07:52.348894       1 reflector.go:255] Listing and watching *v1.DaemonSet from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:320
I0829 12:07:52.348900       1 reflector.go:219] Starting reflector *v1.ReplicaSet (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:347
I0829 12:07:52.348909       1 reflector.go:255] Listing and watching *v1.ReplicaSet from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:347
I0829 12:07:52.348965       1 reflector.go:219] Starting reflector *v1.Job (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:338
I0829 12:07:52.348973       1 reflector.go:255] Listing and watching *v1.Job from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:338
I0829 12:07:52.348994       1 reflector.go:219] Starting reflector *v1.StatefulSet (1h0m0s) from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:356
I0829 12:07:52.349001       1 reflector.go:255] Listing and watching *v1.StatefulSet from k8s.io/autoscaler/cluster-autoscaler/utils/kubernetes/listers.go:356
W0829 12:07:52.364627       1 warnings.go:70] policy/v1beta1 PodDisruptionBudget is deprecated in v1.21+, unavailable in v1.25+; use policy/v1 PodDisruptionBudget
I0829 12:07:52.364817       1 cloud_provider_builder.go:29] Building aws cloud provider.
F0829 12:07:52.364869       1 aws_cloud_provider.go:352] Couldn't open cloud provider configuration /etc/gce.conf: &fs.PathError{Op:"open", Path:"/etc/gce.conf", Err:0x2}
goroutine 30 [running]:
k8s.io/klog/v2.stacks(0x1)
    /gopath/src/k8s.io/autoscaler/cluster-autoscaler/vendor/k8s.io/klog/v2/klog.go:1038 +0x8a
k8s.io/klog/v2.(*loggingT).output(0x611e4e0, 0x3, 0x0, 0xc000565730, 0x0, {0x4d2e584, 0x2}, 0xc0009ea840, 0x0)
    /gopath/src/k8s.io/autoscaler/cluster-autoscaler/vendor/k8s.io/klog/v2/klog.go:987 +0x5fd
k8s.io/klog/v2.(*loggingT).printf(0...
```

Comment (khizunov) [reply charts/cluster-autoscaler/templates/deployment.yaml:66]: @SergiySavarin Thanks for the comment, the fix is in PR https://github.com/kubernetes/autoscaler/pull/5141

**Pull request merged**